gendoc/svcparse/scanner_test.go

```diff
@@ -15,6 +15,7 @@ func cleanStr(s string) string {
 }
 
 func TestBodyDetection(t *testing.T) {
+	t.Skip()
```

Comment:
While going through those tests, I found that that test doesn't actually test anything. As in, it does some stuff and logs a bunch of data, but it doesn't actually check any condition for success or failure. I wanted to reduce the volume of needless prints, so I chose to skip that test, since it can never fail.

@zaquestion any thoughts on my reasoning? Would you have done it differently?